core/src/test/java/org/apache/iceberg/TestSnapshotJson.java

```diff
@@ -56,13 +56,16 @@ public void testJsonConversion() throws IOException {
 
   @Test
   public void testJsonConversionWithoutSchemaId() throws IOException {
-    int snapshotId = 23;
-    Long parentId = null;
-    String manifestList = createManifestListWithManifestFiles(snapshotId, parentId);
-
     Snapshot expected =
-        new BaseSnapshot(
-            0, snapshotId, parentId, System.currentTimeMillis(), null, null, null, manifestList);
+        MetadataTestUtils.buildTestSnapshotWithExampleValues()
```

Comment:
Since this is used to fill base data and then the methods are used to override, should example values be the default behavior? That would make more sense to me.